Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/orientation.h

```diff
@@ -438,8 +438,10 @@ void recursive_orient_volume_ccs( TriangleMesh& tm,
 * inward or outward oriented.
 *
 * @tparam TriangleMesh a model of `FaceListGraph` and `MutableFaceGraph` .
-*                      If `TriangleMesh` has an internal property map for `CGAL::face_index_t`,
-*                      as a named parameter, then it must be initialized.
+*                      If `TriangleMesh`
+  *  has an internal not writable property map
+  *  for `CGAL::face_index_t` and no `face_index_map` is given
+  *  as a named parameter, then the internal one must be initialized; else, it will be.
```

Comment:
```suggestion
*                      If `TriangleMesh` has a non modifiable internal property map
*                      for `CGAL::face_index_t` and no `face_index_map` is given
*                      as a named parameter, then the internal one must be initialized; otherwise, it will be.
```